src/Tes.RunnerCLI/scripts/rootHash.sh

```diff
@@ -0,0 +1,21 @@
+﻿#!/bin/bash
+
+filename=$1
+blocksize=4194304 # 4 MiB
+
+if [ ! -f "$filename" ]; then
+    echo "File not found!"
+    exit 1
+fi
+
+filesize=$(stat -c%s "$filename")
+numblocks=$(((filesize + blocksize - 1) / blocksize))
+
+rm .hashList
+
+for ((i=0; i<$numblocks; i++)); do
+    (dd if="$filename" bs=$blocksize skip=$i count=1 status=none | md5sum | tr -d '\n-' | tr -d ' ') >> .hashList
```

Comment:
do you think users really want a local file created (could just do this with a variable).  If they do, then why is it hidden? I would think either don't use a local file, or, if you do, it shouldn't be a hidden file (starting with a period)